### PR TITLE
X reminder

### LINKED ISSUE #39: Complete Check Command
**Enhancement to #22** 
- add any remaining functions 
- create tests for Check Command

### PR DESCRIPTION
**Pull request by xiaoyu-nus**
This resolves #39

### CODE REVIEW

File: src/main/java/io/xpire/ui/ItemCard.java [hunk 1]
```diff
@@ -15,6 +18,7 @@
 public class ItemCard extends UiPart<Region> {
 
     private static final String FXML = "ItemCard.fxml";
+    private static final String DATE_FORMAT = "dd/MM/yyyy";
```
Comment (JermyTan) [inline src/main/java/io/xpire/ui/ItemCard.java]: Is this being used?

File: src/main/java/io/xpire/logic/parser/SetReminderCommandParser.java [hunk 2]
```diff
@@ -31,7 +34,12 @@ public SetReminderCommand parse(String args) throws ParseException {
                     SetReminderCommand.MESSAGE_USAGE), e);
         }
 
-        String threshold = args.split("\\|")[1];
+        try {
+            threshold = ParserUtil.parseReminderThreshold(args.split("\\|")[1]);
+        } catch (IllegalValueException e) {
+            throw new ParseException(String.format(MESSAGE_INVALID_REMINDER_THRESHOLD,
```
Comment (JermyTan) [inline src/main/java/io/xpire/logic/parser/SetReminderCommandParser.java]: What are the cases where this will be thrown?

Comment (xiaoyu-nus) [reply src/main/java/io/xpire/logic/parser/SetReminderCommandParser.java]: check|1|hello or check|2|-5

File: src/main/java/io/xpire/ui/ItemCard.java (hunk 1 above)
Comment (xiaoyu-nus) [reply src/main/java/io/xpire/ui/ItemCard.java]: Nah will delete that.

File: src/main/java/io/xpire/logic/parser/SetReminderCommandParser.java (hunk 2 above)
Comment (JermyTan) [reply src/main/java/io/xpire/logic/parser/SetReminderCommandParser.java]: Icic, what about set reminder|1 and set reminder|1|?

Comment (xiaoyu-nus) [reply src/main/java/io/xpire/logic/parser/SetReminderCommandParser.java]: Gd point. Not cached. Will fix that.

**Comment by xiaoyu-nus** [comment]
https://flamboyant-golick-d89ac4.netlify.com/userguide
Preview for the website can be found here.

**Comment by JermyTan** [comment]
> https://flamboyant-golick-d89ac4.netlify.com/userguide
> Preview for the website can be found here.

set reminder not in command summary

**Pull request merged**